admin_console/forms/PartnerConfiguration.php

```diff
@@ -633,10 +633,11 @@ public function init()
 		));
 		
 //--------------------------- Recycle Bin Settings ---------------------------
-		
-		$recycleBinRetentionPeriod = new Kaltura_Form_Element_EnumSelect('recycle_bin_retention_period', array('enum' => 'Kaltura_Client_Enum_RecycleBinRetentionPeriod'));
-		$recycleBinRetentionPeriod->setLabel('Recycle Bin retention period');
-		$this->addElements(array($recycleBinRetentionPeriod));
+
+		$this->addElement('text', 'recycle_bin_retention_period', array(
+			'label'			=> 'Recycle Bin retention period',
```

Comment:
missing indication that the value is in days

'Recycle Bin retention period (in Days)'